src/en/manage-cli.md

```diff
@@ -1,554 +1,231 @@
+Title: MAAS CLI
+TODO:  Provide links to definitions of the entities (e.g. fabric, dynamic address range)
+       Decide whether explicit examples are needed
+       Decide whether foldouts should be used
```

Comment:
I think it may be worth adding a TODO (and maybe a new issue):
Add a section that describes how to covert a Resful API call into an executable 'maas' command.